<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogComprehensive</class>
 <widget class="QDialog" name="DialogComprehensive">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>720</width>
    <height>454</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>381</width>
     <height>431</height>
    </rect>
   </property>
   <property name="title">
    <string>综合能力测评</string>
   </property>
   <widget class="QGroupBox" name="groupBox_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>330</y>
      <width>291</width>
      <height>81</height>
     </rect>
    </property>
    <property name="title">
     <string>技能考试</string>
    </property>
    <widget class="QWidget" name="horizontalLayoutWidget_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>271</width>
       <height>41</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>考试类型</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="cbB_Exam"/>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>291</width>
      <height>131</height>
     </rect>
    </property>
    <property name="title">
     <string>竞赛获奖</string>
    </property>
    <widget class="QWidget" name="gridLayoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>271</width>
       <height>101</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>获奖等级</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>竞赛类型</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="cbB_AwardLevel"/>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="cbB_AwardClass"/>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>291</width>
      <height>71</height>
     </rect>
    </property>
    <property name="title">
     <string>志愿服务</string>
    </property>
    <widget class="QWidget" name="horizontalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>19</y>
       <width>271</width>
       <height>41</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>服务次数</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="spinBox_SCount"/>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>260</y>
      <width>291</width>
      <height>61</height>
     </rect>
    </property>
    <property name="title">
     <string>第二学位学习</string>
    </property>
    <widget class="QWidget" name="horizontalLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>271</width>
       <height>31</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>考试科目通过数</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="spinBox_TPCount"/>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QPushButton" name="pbt_AwardAdd">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>80</y>
      <width>61</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>添加</string>
    </property>
    <property name="icon">
     <iconset resource="iamges.qrc">
      <normaloff>:/images/add.ico</normaloff>:/images/add.ico</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="pbt_SCOk">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>190</y>
      <width>61</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>确定</string>
    </property>
    <property name="icon">
     <iconset resource="iamges.qrc">
      <normaloff>:/images/Done.ico</normaloff>:/images/Done.ico</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="pbt_SLPCOk">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>280</y>
      <width>61</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>确定</string>
    </property>
    <property name="icon">
     <iconset resource="iamges.qrc">
      <normaloff>:/images/Done.ico</normaloff>:/images/Done.ico</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="pbt_ECAdd">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>360</y>
      <width>61</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>添加</string>
    </property>
    <property name="icon">
     <iconset resource="iamges.qrc">
      <normaloff>:/images/add.ico</normaloff>:/images/add.ico</iconset>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_6">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>10</y>
     <width>311</width>
     <height>391</height>
    </rect>
   </property>
   <property name="title">
    <string>预览</string>
   </property>
   <widget class="QListView" name="listView">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>291</width>
      <height>361</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="pbt_Close">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>410</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>关闭</string>
   </property>
   <property name="icon">
    <iconset resource="iamges.qrc">
     <normaloff>:/images/706.bmp</normaloff>:/images/706.bmp</iconset>
   </property>
  </widget>
  <widget class="QLabel" name="label_Score">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>410</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="iamges.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pbt_Close</sender>
   <signal>clicked()</signal>
   <receiver>DialogComprehensive</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>666</x>
     <y>423</y>
    </hint>
    <hint type="destinationlabel">
     <x>359</x>
     <y>226</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
